dnf5/commands/replay/replay.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "replay.hpp"
+
+#include "commands/history/arguments.hpp"
+
+#include <dnf5/shared_options.hpp>
+#include <libdnf5/utils/bgettext/bgettext-mark-domain.h>
+
+namespace dnf5 {
+
+void ReplayCommand::set_parent_command() {
+    auto * arg_parser_parent_cmd = get_session().get_argument_parser().get_root_command();
+    auto * arg_parser_this_cmd = get_argument_parser_command();
+    arg_parser_parent_cmd->register_command(arg_parser_this_cmd);
+    arg_parser_parent_cmd->get_group("software_management_commands").register_argument(arg_parser_this_cmd);
+}
+
+void ReplayCommand::set_argument_parser() {
+    auto & cmd = *get_argument_parser_command();
+    cmd.set_description("Replay a transaction that was previously stored to a directory");
```

Comment:
May I ask you why argument requires a directory and not a file?

Reply:
It is because the `--store` option (for any transaction command) outputs a directory which, in addition to the serialized JSON transaction, can contain packages or serialized comps.

I think it is better to be symmetrical: one command produces a directory, other command accepts it.